Dockerfile

```
@@ -0,0 +1,42 @@
+ENV MYSQL_DATABASE_OCM_classroom=OCM_classroom
+ENV MYSQL_DATABASE_OCM_course=OCM_course
+ENV MYSQL_DATABASE_OCM_task=OCM_task
+ENV MYSQL_DATABASE_OCM_submission=OCM_submission
+
+# Copy all microservices into the image
+COPY . /app
+
+# Expose all necessary ports
+
+# Install MySQL and configure it
+RUN apt-get update && \
```

Hadolint (2.12.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)